packages/rainbowkit/locales/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "main": "../dist/locales/index.js",
+  "sideEffects": false
```

Comment:
seems like `sideEffects` false does nothing on external packages, so can probably be removed